integration_tests/models/sql/test_union_exclude_lowercase.sql

```diff
@@ -0,0 +1,4 @@
+select
+    {{ dbt_utils.star(ref("test_union_exclude_base_lowercase"), except=["_dbt_source_relation"]) }}
```

Comment:
As of #624 and #661, you can pass `none` into `source_column_name` to suppress the column from being created, if you'd rather.